frontend/TestRun/DriverMatrixTestRun.svelte

```diff
@@ -24,14 +24,24 @@
     export let runId = "";
     export let buildNumber = -1;
     export let testInfo = {};
+    export let tab = "";
     const dispatch = createEventDispatcher();
     let testRun = undefined;
     let runRefreshInterval;
-    let activityOpen = false;
-    let commentsOpen = false;
-    let issuesOpen = false;
+    let activeTab = tab.toLowerCase() || "details";
     let failedToLoad = false;
 
+    const setActiveTab = (tabName) => {
+        tabName = tabName.toLowerCase();
+        if (tabName !== activeTab) {
+            activeTab = tabName;
+            if (!window.location.pathname.startsWith("/workspace")) {
+                const newUrl = `/tests/${testInfo.test.plugin_name}/${runId}/${tabName}`;
+                history.replaceState({}, "", newUrl);
```

Comment:
You should probably use `pushState` here and handle `statePop` even so that user can go back and forth between tabs using the history buttons

Reply:
I'm not sure about that, usually when jumping between tabs (without reload), feels that go back shouldn't jump tabs - users rather click specific tab instead going back with browser